examples/cargo_without_cmake/Cargo.toml

```diff
@@ -20,8 +20,11 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 cxx = "1.0.75"
 cxx-qt = { path = "../../crates/cxx-qt" }
+# Use `cxx-qt = "0.4"` here instead!
```

Comment:
wonder if these should mention `0.4` here or just `version` or `$version` or something, otherwise it's another thing to update that can go wrong. The book already mentions that you should look on crates.io for the version.

Reply:
We usually go through with a grep and look for each version number when releasing CXX-Qt, so this shouldn't be an issue.

Especially in the "build system" stuff I think it's important to have a line that you can literally copy-and-paste without having to think about it.
People usually don't want to spend lots of time on this.